### PR TITLE
feat(rpc): plumb CDPSession

### PR DESCRIPTION
**Pull request by dgozman**

### CODE REVIEW

File: src/rpc/client/cdpSession.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+ */
+
+import { CDPSessionChannel, CDPSessionInitializer } from '../channels';
+import { ConnectionScope } from './connection';
+import { ChannelOwner } from './channelOwner';
+import { Protocol } from '../../chromium/protocol';
+
+export class CDPSession extends ChannelOwner<CDPSessionChannel, CDPSessionInitializer> {
+  static from(download: CDPSessionChannel): CDPSession {
+    return (download as any)._object;
```
Comment (pavelfeldman) [inline src/rpc/client/cdpSession.ts:24]: download?

File: src/rpc/channels.ts [hunk 2]
```diff
@@ -330,3 +330,18 @@ export type DownloadInitializer = {
...
+// ------------ Chromium ------------
+
+export interface ChromiumBrowserChannel extends BrowserChannel {
+  newBrowserCDPSession(): Promise<CDPSessionChannel>;
+}
+export type ChromiumBrowserInitializer = BrowserInitializer & {};
+
+
+export interface CDPSessionChannel extends Channel {
+  send(params: { method: string, params?: Object }): Promise<Object>;
```
Comment (pavelfeldman) [inline src/rpc/channels.ts:344]: where is event?

**Review by pavelfeldman — APPROVED** [review]

File: src/rpc/client/cdpSession.ts (hunk 1 above)
Comment (dgozman) [reply src/rpc/client/cdpSession.ts:24]: Now you know where I've copied this from :)

File: src/rpc/channels.ts (hunk 2 above)
Comment (dgozman) [reply src/rpc/channels.ts:344]: Fixed.

File: test/channels.spec.js [hunk 3]
```diff
@@ -50,24 +48,48 @@ describe.skip(!CHANNEL)('Channels', function() {
...
+
+  it('should scope browser handles', async({browserType, defaultBrowserOptions}) => {
     const GOLDEN_PRECONDITION = {
-      objects: [ 'chromium', 'browser' ],
+      objects: [ 'chromium', 'chromiumBrowser' ],
       scopes: [
-        { _guid: '', objects: [ 'chromium', 'browser' ] },
-        { _guid: 'browser', objects: [] }
+        { _guid: '', objects: [ 'chromium', 'chromiumBrowser' ] },
+        { _guid: 'chromiumBrowser', objects: [] }
```
Comment (pavelfeldman) [inline test/channels.spec.js:80]: ouch, let's leave it browser?

**Review by pavelfeldman — APPROVED** [review]

**Pull request merged**